content/websockets/gateways.md

```diff
@@ -241,6 +241,13 @@ Occasionally, you may want to have a direct access to the native, **platform-spe
 server: Server;
 ```
 
+Also, you can get namespace instance when you passed `namespace` option to `@WebSocketGateway()` decorator.
+
+```typescript
+@WebSocketServer()
```

Comment:
Don't we need to supply the namespace option here?

Reply:
Oh sorry, I missed applying suggestion.
I applied it.